sensorlib/ReadMe.md

```diff
@@ -0,0 +1,86 @@
+# CARLA Sensor Library
+
+## Overview
+
+The sensor library provides a proxy sensor for the CARLA simulator, exposing additional features.
+
+This new feature is provided to enable vulnerable road user detection using a LIDAR sensor in the context of a
+CARLA-driven simulation. The data is intended to feed into a CARMA V2X system for data fusion with other sensors and
+action by CARMA platform vehicles.
+
+The SimulatedSensor acts as a wrapper to the CARLA Semantic Lidar Sensor, capturing the data and performing
+post-processing to generate a list of detected objects upon request.
+
+Raw sensor observations from the CARLA sensor are analyzed to build a dataset representing the current objects in scene.
+
+## Service Layer
```

Comment:
Please rename this service to CARLA CDASim Adapter. This CARLA library gets used here and in Eric's CARLA CARMA Integration tool so there isn't really one service where this gets used.